style.css

```diff
@@ -0,0 +1,262 @@
+  margin-block-end: 0.01px;
+  margin-block-start: 0.01px;
+  margin-inline-end: 0.01px;
+  margin-inline-start: 0.01px;
+}
+
+p {
+  color: #344563;
+  font-family: "Poppins", sans-serif;
+  text-align: left;
+  font-size: 16px;
+  line-height: 25px;
+  margin-block-end: 0.01px;
+  margin-block-start: 0.01px;
+  margin-inline-end: 0.01px;
+  margin-inline-start: 0.01px;
+}
+
+button {
+  font-size: 17px;
+  color: #396df2;
+  background: #fff;
+  border: none;
+  font-weight: 550;
+}
+
+.page-wrapper {
+  background: #e5e5e5;
+  width: 380px;
+  margin: 0 auto;
+  justify-content: center;
+}
+
+.work {
+  display: grid;
+  justify-content: space-between;
+  align-content: center;
+  grid-template-rows: auto auto auto auto;
+  padding: 10px;
+  row-gap: 100px;
+}
+
+.card-work,
+.about-myself,
+.contact-form {
+  background: #fff;
+  width: 330px;
+  padding: 15px;
+  border-radius: 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+
+.snapshoot img {
+  width: 100%;
+}
+
+.tags {
+  display: flex;
+  flex-direction: row;
+  justify-content: left;
+}
+
+.tags li {
+  padding: 5px 10px;
+  margin-right: 20px;
+  border-radius: 8px;
+  background: #ebebff;
+  margin-block-end: 0.01px;
+  margin-block-start: 0.01px;
+  margin-inline-end: 10px;
+  margin-inline-start: 0.01px;
+}
+
+.normal-button {
+  width: fit-content;
+  border: 1px solid #6070ff;
+  margin-top: 30px;
+  border-radius: 8px;
+  padding: 15px 20px;
+}
+
+.about-myself {
+  padding-top: 100px;
+  width: 92%;
+  margin: 100px auto;
+  border-radius: 0 70px 0 0;
+}
+
+.let-connect {
+  padding-top: 10px;
+  color: #7f8cff;
+  font-size: 15px;
+  font-weight: bold;
+}
+
+.social {
+  display: flex;
+  flex-direction: column;
+}
+
+.social img {
+  padding-top: 25px;
+  padding-left: 14px;
+  margin-block-end: 0.01px;
+  margin-block-start: 0.01px;
+  margin-inline-end: 0.01px;
+  margin-inline-start: 0.01px;
+}
+
+.social-icons {
+  display: flex;
+  flex-direction: row;
+  justify-content: start;
+  list-style-type: none;
+  padding-inline-start: 1px;
+  margin-block-start: 0.01em;
+}
+
+.contact-text {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  text-align: center;
+  padding: 10px;
+}
+
+.contact-me {
+  background-image: url(./images/Contact_me.png);
+}
+
+.contact-text p {
+  text-align: center;
+}
+
+.text-form {
+  font-family: "Poppins", sans-serif;
+  color: #172b4d;
+  font-size: 17px;
+  padding: 10px;
+  border-radius: 8px;
+  border: 1px solid #fff;
+  background: #fff;
+}
+
+/* FIRST PROJECT STYLING */
+
+.container {
+  max-width: 375px;
+  height: 100vh;
+  margin: 0 auto;
+}
+
+a {
+  text-decoration: none;
+}
+
+h1 {
+  font-family: "Poppins", sans-serif;
+  font-weight: 900;
+  color: #172b4d;
+  line-height: 50px;
+  font-size: 40px;
+  margin-top: 60px;
+}
+
+/* p {
+  color: #344563;
+  font-family: "Poppins", sans-serif;
+  text-align: left;
+  font-size: 16px;
+  line-height: 23px;
+} */
```

Comment:
- [ ] Please remove commented code from your codebase that you are not using nor is not a meaningful comment